src/Items/ItemSlab.h

```diff
@@ -73,20 +58,20 @@ class cItemSlabHandler:
 				}
 				return true;
 			}
+			return false;
```

Comment:
Just because you can't merge the slabs doesn't mean they can't be placed at all.

Reply:
This is what the old code did, so I'm not changing it :)

Reply:
No, it didn't. You are changing it...

In the original code it was possible for the first `if` condition to be true and the code could still fall through to checking the `AddFaceDirection` block. Because of this return statement, it is no longer possible.

Reply:
Right, the extra return. That got in the way somehow.